deepcell/applications/nuclear_segmentation.py

```diff
@@ -94,40 +107,71 @@ class NuclearSegmentation(Application):
         model (tf.keras.Model): The model to load. If ``None``,
             a pre-trained model will be downloaded.
     """
-
-    #: Metadata for the dataset used to train the model
-    dataset_metadata = {
-        'name': 'general_nuclear_train_large',
-        'other': 'Pooled nuclear data from HEK293, HeLa-S3, NIH-3T3, and RAW264.7 cells.'
-    }
-
-    #: Metadata for the model and training process
-    model_metadata = MODEL_METADATA
-
-    def __init__(self, model=None,
-                 preprocessing_fn=histogram_normalization,
-                 postprocessing_fn=deep_watershed):
-
-        if model is None:
-            cache_subdir = 'models'
-            model_dir = Path.home() / ".deepcell" / "models"
-            archive_path = fetch_data(
-                asset_key=MODEL_KEY,
-                cache_subdir=cache_subdir,
-                file_hash=MODEL_HASH
-            )
-            extract_archive(archive_path, model_dir)
-            model_path = model_dir / MODEL_NAME
-            model = tf.keras.models.load_model(model_path)
+    def __init__(self,
+                 model,
+                 model_mpp=0.65,
+                 radius=10,
+                 maxima_threshold=0.1,
+                 interior_threshold=0.01,
+                 exclude_border=False,
+                 small_objects_threshold=0,
+                 min_distance=10
+                 ):
+
+        self.postprocess_kwargs = {
+            'radius': radius,
+            'maxima_threshold': maxima_threshold,
+            'interior_threshold': interior_threshold,
+            'exclude_border': exclude_border,
+            'small_objects_threshold': small_objects_threshold,
+            'min_distance': min_distance
+        }
 
         super().__init__(
             model,
             model_image_shape=model.input_shape[1:],
-            model_mpp=MODEL_MPP,
-            preprocessing_fn=preprocessing_fn,
-            postprocessing_fn=postprocessing_fn,
-            dataset_metadata=self.dataset_metadata,
-            model_metadata=self.model_metadata)
+            model_mpp=model_mpp,
+            preprocessing_fn=histogram_normalization,
+            postprocessing_fn=deep_watershed)
+
+    @classmethod
+    def from_version(cls, version='1.1'):
+        """Load a specified version of the model
+
+        1.1: Updates to the nuclear segmentation model released in July 2024
+        1.0: Original nuclear segmentation model released with the September 2023 Caliban preprint
+
+        Args:
+            version (str, optional): Defaults to '1.1'.
```

Comment:
This *might* make the docs build happy - honestly it doesn't matter though, IMO the docstring parser is just overly picky.

```suggestion
            version (:obj:`str`, optional): Defaults to '1.1'.
```